setup.py

```diff
@@ -42,6 +42,7 @@ def read(fname):
     'pathlib2',
     'responses',
     'boto3',
+    'python-jose',
```

Comment:
What is this and why do we need it?

Reply:
pip threw some error about resolving dependencies that when I ran the test suite on MacOS Catalina. I'll try removing it and report back with the specific error.

Reply:
When running the tests locally on MacOS Catalina, I get this error:

```
ImportError while importing test module 'smart_open/tests/test_smart_open.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
python3.7/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
smart_open/tests/test_smart_open.py:21: in <module>
    from moto import mock_s3
.env/lib/python3.7/site-packages/moto/__init__.py:14: in <module>
    from .cognitoidp import mock_cognitoidp, mock_cognitoidp_deprecated  # noqa
.env/lib/python3.7/site-packages/moto/cognitoidp/__init__.py:2: in <module>
    from .models import cognitoidp_backends
.env/lib/python3.7/site-packages/moto/cognitoidp/models.py:13: in <module>
    from jose import jws
E   ModuleNotFoundError: No module named 'jose'
```

installing python-jose resolves it. The error looks like it tracks back to the `mock_s3` package

Reply:
I think that's a problem with `moto`. Please open a ticket with them and let them know they're missing a dependency.

We shouldn't have to deal with it in our code.

Reply:
Removed it